### ISSUE TITLE
Separation of errors about authorization vs authentication

### ISSUE THREAD

**Issue by mrquincle** [crownstone/crownstone-cloud#2]
Currently trying to obtain a sphere with an incorrect id, for example `id 0` leads to 401 errors:

http://crownstone-cloud.herokuapp.com/explorer/#!/Sphere/Sphere_findById

This is exactly the same error as when login doesn't succeed.

It is recommended to use 403 errors for authorization / application logic: 

- See [stackoverflow](http://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses)
- This will also make it easier to see how a connection fails to the cloud.

**Comment by AlexDM0** [comment]
I agree. We'll have to backtrack where these are generated. Odds are that they're loopback native..
